patches/chrome-browser-themes-theme_service_factory.cc.patch

```diff
@@ -10,3 +10,11 @@ index caec77bbe3ba59b030622d3a9ff4fb1edd0a476d..b1b48ed7fb8022d6377594754ce1f300
  #if defined(OS_LINUX) && !defined(OS_CHROMEOS)
    using ThemeService = ThemeServiceAuraLinux;
  #endif
+@@ -108,6 +109,7 @@ void ThemeServiceFactory::RegisterProfilePrefs(
+ 
+ content::BrowserContext* ThemeServiceFactory::GetBrowserContextToUse(
+     content::BrowserContext* context) const {
++  BRAVE_THEMESERVICEFACTORY_GETBROWSERCONTEXTTOUSE
```

Comment:
Can we redefine the method in `chromium_src` instead?

Reply:
@mkarolin Yes. I did :) PTAL.